src/test/java/seedu/address/model/person/MatriculationNumberTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class MatriculationNumberTest {
```

Comment:
Same as before! Also, the tests are very organised! Great job. :yellow_heart:

Reply:
I think for test classes, there shouldn't be a need for a description 😄 . I'm following the earlier standards presented in EmailTest and PhoneTest. Thank you 👍